suite/tests/CMakeLists.txt

```diff
@@ -4164,6 +4164,33 @@ if (BUILD_CLIENTS)
 
 endif (BUILD_CLIENTS)
 
+
+###########################################################################
+# drpt2trace tests
+if (BUILD_CLIENTS)
+  if (HAVE_LIBIPT)
+    set(drpt2trace_args "-stats"
+      "-raw_pt"
+      "${PROJECT_SOURCE_DIR}/clients/drcachesim/drpt2trace/test_simple.tracedir/pt.bin"
+      "-primary_sb"
+      "${PROJECT_SOURCE_DIR}/clients/drcachesim/drpt2trace/test_simple.tracedir/sideband.pevent"
+      "-pt_cpu_f" "6"
```

Comment:
Add a comment to say that these option values were obtained using the libipt get-opt script.

Reply:
Done.